lib/Service/CardService.php

```diff
@@ -77,6 +77,7 @@ public function __construct(
 		LoggerInterface $logger,
 		IRequest $request,
 		CardServiceValidator $cardServiceValidator,
+		private AssignmentService $assignmentService,
```

Comment:
If we touch the constructor we can move all other properties to property promition as well